### PR TITLE
[DOCFIX] Improve POSIX API Doc

### PR DESCRIPTION
**Pull request by ssz1997**
- Add in a few colons to make the lines more readable. 
- Fix a few typos.
- For the "available Linux mount options" man page on line #222, the original link returns 404. I found a related man page but not entirely sure this is the correct one. Let me know if the link is wrong.

### CODE REVIEW

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -219,13 +219,13 @@ value of `-o`.
 The `-o [mount options]` must follow the `mount` command.
 
 Different versions of `libfuse` and `osxfuse` may support different mount options.
-The available Linux mount options are listed [here](http://man7.org/linux/man-pages/man8/mount.fuse.8.html).
+The available Linux mount options are listed [here](http://man7.org/linux/man-pages/man8/mount.8.html).
```
Comment (HelloHorizon) [inline docs/en/api/POSIX-API.md:222]: hey @LuQQiu, can you confirm this is the link we want for the fuse?

**Review by HelloHorizon — COMMENTED** [review]
@ssz1997 Thanks for the improvement!

File: docs/en/api/POSIX-API.md (hunk 1 above)
Comment (HelloHorizon) [reply docs/en/api/POSIX-API.md:222]: I found two potential reference for mount option for fuse:
`https://man7.org/linux/man-pages/man8/mount.fuse3.8.html`
`https://man7.org/linux/man-pages/man8/mount.fuse3.8@@libfuse.html`

Comment (ssz1997) [reply docs/en/api/POSIX-API.md:222]: I think they are the correct ones as they are fuse-related. I'm updating it now.

**Comment by ssz1997** [comment]
alluxio-bot, check this please

**Review by HelloHorizon — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**